src/components/SideBar.js

```diff
@@ -0,0 +1,65 @@
+import DocumentTree from "./DocumentTree.js"
+import { push } from "../utils/router.js"
+
+export default function SideBar({ $target, initialState, onAdd, onDelete}) {
+    const $sideBar = document.createElement('div')
+    $sideBar.className = 'sideBar'
+
+    $target.appendChild($sideBar)
+
+    const $header = document.createElement('span')
+    $header.innerHTML= `<img class="notion-icon" src="../../images/notion.png" /><h3>hyunjoo의 Notion</h3>`
+    $header.className = 'document-header'
```

Comment:
요 부분 아까 제안드렸던 dom 함수를 사용하면 이렇게 js로 변경해볼 수 있어요!

```suggestion
const $headerImg = createCustomElement({
    tag: 'img',
    props: { className: 'notion-icon', src: '../../images/notion.png' },
  });
    
    const $headerTitle = createCustomElement({
    tag: 'h3'
  });
  
    const $header = createCustomElement({
    tag: 'span',
    props: { className: 'document-header' },
    children: [$headerImg, $headerTitle],
  });

```

코드가 조금 길어지긴 하지만, string 작성으로 인해
발견되지 않는 오류가 일어날 가능성을 줄여줄 수 있습니다 ㅎㅎ